src/_layouts/home.html

```diff
@@ -3,24 +3,28 @@
         <title>{{ page.title }}</title>
         <link rel="stylesheet" href="/assets/css/styles.css" />
         <link rel="stylesheet" href="/assets/css/splash.css" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     </head>
     <body>
         <header class="header">
             <a href="#logo"><p>LIGHTNING HACKS</p></a>
         </header>
         <div class="splash" id="logo">
             <div class="splash--item">
-                {% include logo_yellow.html id="logo" animate=true width='50vw' %}
+                {% include logo_yellow.html id="logo" animate=true width='100%' %}
             </div>
             <div class="splash--item">
-                <div class="title"> 
-                    <p>Uma</p>
-                    <h3>INICIATIVA</h3>
-                    <p>de <b>hackers</b> para <b>hackers</b></p>
+                <div class="title">
+                    <p>Apresentações relâmpago</p>
+                    <p>que quebram paradigmas</p>
                 </div>
             </div>
         </div>
         {{ content }}
         {% asset bundle.js %}
+        <footer class="home--footer">
```

Comment:
Strange to use `home` CSS over here. Even though it's the home layout, we didn't use `home` anywhere... Since we're probably going to use this footer in every other place, maybe we should make it an `include` and create its own CSS.